configuring/shared-resources-csi-driver.adoc

```diff
@@ -0,0 +1,26 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="shared-resource-csi-driver"]
+= Shared Resource CSI Driver Operator
+include::_attributes/common-attributes.adoc[]
+:context: shared-resource-csi-driver
+
+toc::[]
+
+
+[role="_abstract"]
+As a cluster administrator, you can use the Shared Resource CSI Driver in {builds-title} to provision inline ephemeral volumes that contain the contents of `Secret` or `ConfigMap` objects. This way, pods and other Kubernetes types that expose volume mounts, and {ocp-product-title} Builds can securely use the contents of those objects across potentially any namespace in the cluster. To accomplish this, there are currently two types of shared resources: a `SharedSecret` custom resource for `Secret` objects, and a `SharedConfigMap` custom resource for `ConfigMap` objects.
+
+
+include::modules/persistent-storage-csi-about.adoc[leveloffset=+1]
+
+include::modules/ephemeral-storage-sharing-secrets-across-namespaces.adoc[leveloffset=+1]
+
+include::modules/ephemeral-storage-using-a-sharedsecrets-resource-in-a-pod.adoc[leveloffset=+1]
```

Comment:
This is creating the following error in a build:
```
Unresolved directive in shared-resources-csi-driver.adoc - include::modules/ephemeral-storage-using-a-sharedsecrets-resource-in-a-pod.adoc[leveloffset=+1]
```
and I can't figure out what file it's supposed to be referring to, in case it was a typo like the one on line 16.